x-pack/plugins/reporting/server/export_types/printable_pdf/types.d.ts

```diff
@@ -10,13 +10,22 @@ import { BaseParams, BasePayload } from '../../types';
 
 interface BaseParamsPDF {
   layout: LayoutParams;
-  forceNow?: string;
-  // TODO: Add comment explaining this field
   relativeUrls: string[];
+  isDeprecated?: boolean;
 }
 
 // Job params: structure of incoming user request data, after being parsed from RISON
 export type JobParamsPDF = BaseParamsPDF & BaseParams;
 
 // Job payload: structure of stored job data provided by create_job
-export type TaskPayloadPDF = BaseParamsPDF & BasePayload;
+export interface TaskPayloadPDF extends BasePayload {
+  layout: LayoutParams;
+  forceNow?: string;
+  objects: Array<{ relativeUrl: string }>;
```

Comment:
Note, previously in the `master` branch, this `TaskPayloadPDF` field was called `relativeUrls`, which matches the interface data in the incoming `BaseParamsPDF` data. This was done to ease TypeScript conversion of the PDF export type: the compatibility shim was not converted into TypeScript until now.